modules/KalturaSupport/resources/mw.id3Tag.js

```diff
@@ -71,17 +71,22 @@
         parseTag: function(tag){
             var time;
             if ( tag ) {
-                time = tag.timestamp / 1000;
+                // on IE the tag needs to be parsed
+                time = JSON.parse(tag).timestamp / 1000;
             } else {
                 mw.log("id3Tag plugin :: ERROR parsing tag.");
             }
             if(time) {
                 var d = new Date(0); // The 0 there is the key, which sets the date to the epoch
                 d.setUTCSeconds(time);
-                this.log("Update time from id3 tag: " + d.toUTCString());
+	            this.log("Update time from id3 tag: " + d.toUTCString());
                 this.updatedTime = time;
                 this.counter = 0; //reset time update interval counter
                 this.getPlayer().LiveCurrentTime = time;
+                if(this.getPlayer().isMulticast){
+	                //set the current time for MC from the ID3 tag and not from the SL element that has always 0
+	                this.getPlayer().currentTime = time
```

Comment:
this looks very odd....  places that uses currentTime should use LiveCurrentTime instead.